package.json

```diff
@@ -4,7 +4,9 @@
     "name": "Steve Kellock"
   },
   "ava": {
-    "require": ["./out-test/main/test/setup.js"]
+    "require": [
+      "./out-test/main/test/setup.js"
+    ]
```

Comment:
Ya, haha, this is because `npm` reformats it.  I use prettier to format `json`.  I'll put a lint step in there for it... no worries dude!  It's fine.